pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/util/model/ParticipantUtilTest.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.dsm.util.model;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.broadinstitute.dsm.util.ParticipantUtil;
+import org.junit.Test;
+
+public class ParticipantUtilTest {
+
+    @Test
```

Comment:
Ok, I thought in case one doesn't pass (invalid or valid) it's easier to look at green and red tests and see which one is not passing, but I will move them into one test if it looks better that way